src/main/java/org/junitpioneer/jupiter/TempDirectory.java

```diff
@@ -345,6 +346,7 @@ private void tryToDeleteOnExit(Path path) {
 				path.toFile().deleteOnExit();
 			}
 			catch (UnsupportedOperationException ignore) {
+				// this exception shall be ignored by purpose
```

Comment:
What purpose? Shouldn't that be included in the comment?

Reply:
> What purpose? Shouldn't that be included in the comment?

To be honest I'm not sure what the exactly purpose is, but as the method is only called in on method which creates an own exception, I worte that down